examples/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

Comment:
Each example already has its own `CMakeLists`. Would it be possible to remove this top-level `CMakeLists`, which has duplicate information, and instead, in the test, iterate over all the directories inside `examples`?